### DISCUSSION TITLE
Which repo to use while trying to use customAuth for flutter (jwt)

### DISCUSSION THREAD

**Discussion by nerzhul250** [Web3Auth/Web3Auth#643] [Help]
Hello, just a quick question

Found two repos that implement customAuth for flutter, don´t know which repo to use (my guess is the one from web3Auth, but not completely sure)

https://github.com/torusresearch/customauth-flutter-sdk

https://github.com/Web3Auth/web3auth-flutter-sdk

thanks in advance!

**Comment by himanshuchawla009** [comment]
https://github.com/Web3Auth/web3auth-flutter-sdk  is the latest
You can read the docs about it [here](https://web3auth.io/docs/sdk/flutter/).
